src/test/java/Homework17.java

```diff
@@ -56,11 +56,12 @@ public static void addSongToPlaylistTest () throws InterruptedException {
         morningJamLink.click();
 
         //Validate "Reactor" song is added
-        
-
-
-
-
+        Thread.sleep(3000);
+        WebElement morningJamPlaylist = driver.findElement(By.cssSelector("[href='#!/playlist/18754']"));
+        morningJamPlaylist.click();
+        WebElement reactorSongIsPresent = driver.findElement(By.xpath("//*[@id=\"playlistWrapper\"]/div/div/div[1]/table/tr[13]/td[2]"));
```

Comment:
if you are beginner does not know who to write the xpath then you can relay on the xpath that will be provided by chrome browser. Right click on element that you are interested in then click on Inspect. Now right click on the html code of the element Copy > Copy Xpath. Now you have the xpath in the clipboard for you.

For example, here is the locator (Copy > Copy Xpath):  `//*[@id=\"playlistWrapper\"]/div/div/div[1]/table/tr[13]/td[2]`

**BUT I recommend not using plugins or copying the xpath** from the page code using a browser or web developer.

For example, here is the correct locator:  `//a[@class='songs']`